### PR TITLE
Assignment lesson_1_6

### PR DESCRIPTION
**Pull request by tungphamwd**

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from "react";
+
+const InputWithLabel = ({ todoTitle, handleTitleChange, children }) => {
+  const inputRef = useRef(null);
+  useEffect(() => {
+    inputRef.current.focus();
+  },[])
```
Comment (riousenkai) [inline src/InputWithLabel.js:7]: this shouldn't have a dependency array

File: src/TodoListItem.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-const TodoListItem = ({todo}) => {
+const TodoListItem = ({todo, onRemoveTodo}) => {
     return (
-        <li>{todo.title}</li>
+        <li>{todo.title}<button onClick={() => onRemoveTodo(todo.id)}>Delete</button></li>
```
Comment (riousenkai) [inline src/TodoListItem.js:3]: In the front end there are is no space between the todo title and the remove button. Add a space in between so they're not together.

**Review by riousenkai — APPROVED** [review]
I'm approving but please resolve the comments I made in your most recent assignment.

File: src/InputWithLabel.js (hunk 1 above)
Comment (tungphamwd) [reply src/InputWithLabel.js:7]: Thank you. I'll fix that.

File: src/TodoListItem.js (hunk 2 above)
Comment (tungphamwd) [reply src/TodoListItem.js:3]: Thank you. I'll add more space.

**Pull request merged**